drivers/netdev_ieee802154_submac/netdev_ieee802154_submac.c

```diff
@@ -61,6 +61,9 @@ static int _get(netdev_t *netdev, netopt_t opt, void *value, size_t max_len)
         case NETOPT_STATE:
             *((netopt_state_t*) value) = _get_submac_state(submac);
             return 0;
+        case NETOPT_TX_RETRIES_NEEDED:
+            *((int8_t*) value) = netdev_submac->retrans;
```

Comment:
I noticed something else: The NETOPT_TX_RETRIES_NEEDED opt expects an `uint8_t` (see [NETOPT documentation](https://doc.riot-os.org/group__net__netopt.html#gga19e30424c1ab107c9c84dc0cb29d9906a860b21d7335e516435b0762e246bdfb5)).
I suggest to do something like:
```suggestion
            if (netdev_submac->retrans < 0) {
                return -ENOTSUP;
            }
            *(uint8_t*) value) = netdev_submac->retrans;
```